### DISCUSSION TITLE
增加只识别数字类型

### DISCUSSION THREAD

**Discussion by asdqwed** [hiroi-sora/Umi-OCR#634] [Ideas]
我查阅了API 文档能否增加一个选项只识别数字OCR？这样对于只需要识别数字的使用场景精确率会高很多，速度还能快很多。还能避免误识别，实用性还蛮高的。

**Comment by hiroi-sora** [comment]
PaddleOCR项目组没有发布纯数字模型库，所以只能自己训练模型。我们暂时没有训练模型的计划。
